source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po

```diff
@@ -364,7 +364,7 @@ msgid ""
 "``[SamplerN]``, ``[Master]``, ... groups."
 msgstr ""
 "De volgende uitbreidingen voegen enkele mogelijkheden toe aan "
-"``ControlPotMeter`` controls (volume, :term:`crossfader`, ...). Gebruik ze "
+"``Draaiknoppen`` (volume, :term:`crossfader`, ...). Gebruik ze "
```

Comment:
`ControlPotMeter` is a technical term that should not be translated.

Reply:
Hmm, not necessarily a term that needs to be explained in the glossary. The explanation is at the top of the Mixxx controls page, and aren't all ControlPotMeter mentions refs?

Reply:
you're right, I had to check the files in the manual and on my pc to see that it should be in a box, therefor it needs to be the English term.